app/src/main/java/hm/orz/chaos114/android/tumekyouen/app/StageGetDialog.java

```diff
@@ -9,98 +8,100 @@
 import android.widget.CompoundButton;
 import android.widget.EditText;
 
+import hm.orz.chaos114.android.tumekyouen.R;
+
 public class StageGetDialog extends ValidationDialog {
 
-	/** ステージ数入力領域 */
-	private EditText numberEdit;
+    /** ステージ数入力領域 */
+    private EditText numberEdit;
 
-	/** 全件チェックボックス */
-	private CheckBox allCheckBox;
+    /** 全件チェックボックス */
+    private CheckBox allCheckBox;
 
-	/** 成功時のリスナー */
-	public interface OnSuccessListener {
-		void onSuccess(int count);
-	}
+    /**
+     * コンストラクタ。
+     *
+     * @param context         コンテキスト
+     * @param successListener 成功時のリスナー
+     * @param cancelListener  キャンセル時のリスナー
+     */
+    public StageGetDialog(Context context,
+                          final OnSuccessListener successListener,
+                          OnCancelListener cancelListener) {
+        super(context);
 
-	/**
-	 * コンストラクタ。
-	 * 
-	 * @param context コンテキスト
-	 * @param successListener 成功時のリスナー
-	 * @param cancelListener キャンセル時のリスナー
-	 */
-	public StageGetDialog(Context context,
-			final OnSuccessListener successListener,
-			OnCancelListener cancelListener) {
-		super(context);
+        // viewの設定
+        LayoutInflater layoutInflater = getLayoutInflater();
+        View view = layoutInflater.inflate(R.layout.stage_get_dialog, null);

```

Comment:
WARNING: [InflateParams] Avoid passing `null` as the view root (needed to resolve layout parameters on the inflated layout's root element)
 When inflating a layout, avoid passing in null as the parent view, since otherwise any layout parameters on the root of the inflated layout will be ignored.